src/GoogleMyBusiness.php

```diff
@@ -69,10 +70,10 @@ class GoogleMyBusiness extends Google_Service
   public function __construct(Google_Client $client)
   {
     parent::__construct($client);
-    $this->rootUrl = 'https://mybusiness.googleapis.com/';
+    $this->rootUrl = 'https://mybusinessaccountmanagement.googleapis.com/';
     $this->servicePath = '';
     $this->version = 'v4';
-    $this->serviceName = 'mybusiness';
+    $this->serviceName = 'mybusinessaccountmanagement';
```

Comment:
This is wrong! we cannot change the root url, how the other methods will work? A new rootUrl and service name must be created to support the new API endpoint and service name respectively